src/test/java/HomeWork21.java

```diff
@@ -0,0 +1,56 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class HomeWork21 extends BaseTest {
+    String newPlaylistName = "Sample Edited Playlist";
+
+    @Test
+    public void renamePlaylist() throws InterruptedException {
+
+
+        String updatedPlaylistSuccessMsg = "Updated playlist \"Sample Edited Playlist.\"";
+
+        //Log in to Koel
+        provideEmail("jason.fenstermaker1234@testpro.io");
+        providePassword("Testpro.io2");
+        loginToKoel();
+
+        //Double click
+        doubleClickPlaylist();
+
+        //Enter new name
+        enterNewName();
+
+        //assertion
+        Assert.assertEquals(getRenamePlayListSuccessMsg(), updatedPlaylistSuccessMsg);
+
+
+    }
```

Comment:
**Use in test methods, the following logical split:   GIVEN – WHEN - THEN**
   
   For example
   
![Clean_Test](https://github.com/jasonfenster/A55-QA-Automation-Framework/assets/104761482/0e03be0a-7957-432c-b36c-2b4fa0c06318)